### ISSUE TITLE
Coversion of Wav2vec2 model to TFWav2vec2 model

### ISSUE THREAD

**Issue by harveenchadha** [huggingface/transformers#13180]
Hi,

I trained model using fairseq toolkit.

I have successfully converted the model from fairseq to huggingface .bin model.

How can I convert to pure pytorch (.pt) and tensorflow (.h5) format.

Are there any scripts for that?

**Comment by NielsRogge** [comment]
Yes there'a script for that, you can find it [here](https://github.com/huggingface/transformers/blob/master/src/transformers/convert_pytorch_checkpoint_to_tf2.py).

**Comment by patrickvonplaten** [comment]
Hey @harveenchadha, to convert from PT to TF, you can just do:

```python
from transformers import TFWav2Vec2Model

model = TFWav2Vec2Model.from_pretrained("<path/to/hf.bin model folder>", from_pt=True)
model.save_pretrained("<path/to/save/.h5>")
```

**Comment by patrickvonplaten** [comment]
Also note that (`.pt`) and (`.bin`) is the same format in most cases as far as I understand: https://stackoverflow.com/questions/57245332/what-are-the-difference-between-bin-and-pt-pytorch-saved-model-types#:~:text=1%20Answer&text=There%20is%20no%20difference%20as,torch%20can%20read%20either%20.
